### PR TITLE
Added proxy_idp_auth_test.sh

### PR DESCRIPTION
**Pull request by vyskocilpavel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,21 @@ paths=""
...
+* Attributes to be filled:
+<pre>
+# The url of tested SP
+# For example :https://aai-playground.ics.muni.cz/simplesaml/nagios_check.php?proxy_idp=cesnet
+testSite=""
+
+loginSite="https://idp2.ics.muni.cz/idp/Authn/UserPassword"
+
+# Fill in UCO as number
+uco=
```
Comment (tauceti2) [inline README.md]: Please rename it to login. The probe should be generic.

File: README.md [hunk 2]
```diff
@@ -17,6 +17,21 @@ paths=""
...
 </pre>
 
+### proxy_idp_auth_test.sh
+* Attributes to be filled:
+<pre>
+# The url of tested SP
+# For example :https://aai-playground.ics.muni.cz/simplesaml/nagios_check.php?proxy_idp=cesnet
+testSite=""
+
+loginSite="https://idp2.ics.muni.cz/idp/Authn/UserPassword"
```
Comment (tauceti2) [inline README.md]: Also use idp2.ics.muni.cz as an example and provide a line with description what is the loginSite variable.

File: README.md (hunk 1 above)
Comment (vyskocilpavel) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (vyskocilpavel) [reply README.md]: Done

**Pull request merged**